OpenKh.Patcher/Metadata.cs

```diff
@@ -0,0 +1,60 @@
+using OpenKh.Kh2;
+using System.Collections.Generic;
+using System.IO;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace OpenKh.Patcher
+{
+    public class Metadata
```

Comment:
This file / class should definitely be turned into a document somehow.

Reply:
I know... I kind-of expect that someone from the community would come up with a good tutorial or documentation on how to manage the `mod.yml`. I honestly do not feel do not write it for few reasons. But I imagine it will happen as a separate PR at some point...?

Reply:
I was thinking of something like Doxygen to auto-generate documentation for that class. Because this file is the pillar of the whole mod manager. If something changes in there, mods may won't work properly anymore. But i don't see that in the scope of this PR, just wanted to trigger a discussion.